src/sentry/sentry_metrics/configuration.py

```diff
@@ -41,27 +41,27 @@ def _register_ingest_config(config: MetricsIngestConfiguration) -> None:
     _METRICS_INGEST_CONFIG_BY_USE_CASE[config.use_case_id] = config
 
 
-def get_ingest_config(use_case_key: UseCaseKey) -> MetricsIngestConfiguration:
-    if len(_METRICS_INGEST_CONFIG_BY_USE_CASE) == 0:
-        _register_ingest_config(
-            MetricsIngestConfiguration(
-                db_model=DbKey.STRING_INDEXER,
-                input_topic=settings.KAFKA_INGEST_METRICS,
-                output_topic=settings.KAFKA_SNUBA_METRICS,
-                use_case_id=UseCaseKey.RELEASE_HEALTH,
-                internal_metrics_tag="release-health",
-                writes_limiter_cluster_options=settings.SENTRY_METRICS_INDEXER_WRITES_LIMITER_OPTIONS,
-            )
-        )
-        _register_ingest_config(
-            MetricsIngestConfiguration(
-                db_model=DbKey.PERF_STRING_INDEXER,
-                input_topic=settings.KAFKA_INGEST_PERFORMANCE_METRICS,
-                output_topic=settings.KAFKA_SNUBA_GENERIC_METRICS,
-                use_case_id=UseCaseKey.PERFORMANCE,
-                internal_metrics_tag="perf",
-                writes_limiter_cluster_options=settings.SENTRY_METRICS_INDEXER_WRITES_LIMITER_OPTIONS_PERFORMANCE,
-            )
-        )
+_register_ingest_config(
```

Comment:
Tactical advice. While I think it is a good idea, I would hold this change out of this PR as it is going to miserably conflict with this 
https://github.com/getsentry/sentry/pull/37745/files

Which is making fundamental changes to this file.